stable/admin/values.yaml

```diff
@@ -226,4 +226,20 @@ admin:
 
   # These annotations will pass throgh to the pod as supplied, useful for integrating 3rd party products such as Vault.
   podAnnotations: {}
-    # vault.hashicorp.com/agent-inject: true
\ No newline at end of file
+    # vault.hashicorp.com/agent-inject: true
+
+nuocollector:
+  enabled: false
```

Comment:
Are we disabling by default?   I thought we were going to enable once the sidecar container was publicly available.

Reply:
If we enable it by default, then this means that on every installation we'll be collecting stats and dropping them on the floor, if `insights` is not installed. I believe it's better if we keep the default to `false` and enable it only if the iser have plans to install Insights or any other monitoring solution.

@jleslie85 , @vegichan What do you think?

Reply:
I prefer to pick the simpler default. Aka: don't enable stuff that people might not need and is not essential. In this case, I prefer `false`.

Reply:
With the old model of not having default collectors hardcoded into the container.   You could enable at any time.  So by using `enabled=true` you installed nuocollector but no plugins were installed.   So no additonal overhead and you could add the plugins at any time.    With the move to hardcoding the default plugins,  you are stuck to deciding at deployment time that you want to use nuocollector.  If you decide later to enable you have to shutdown  pods and redeploy them.

Reply:
yeah. You change a value, upgrade the helm deployments and you get a rolling upgrade. It's a trivial process.

Reply:
It's not trivial in production when you have active connections / transactions that will get aborted. and the application logic might not be written to handle reconnect all that well.

Reply:
I think our message to users must be: "Insights should be enabled on _all_ databases - and only disabled if there are specific reasons."

So accordingly, I believe our default should reflect that - ie 'insights.enabled: true'.

Since engineering and support also find the output of insights to be very valuable in diagnosing issues, we should not be encouraging customers to deploy databases without insights.

I fully expect everyone in engineering, support - and customer organisations - will want all PROD databases to have insights enabled, therefore the overhead of insights needs to be acceptable for production.
If it isn't, then we need to address that, rather than disabling insights and ignoring the performance problems.

Reply:
We could have a default output to the log that can be override when deploying insights or some other server-side component.  The issue I see with defaulting to stdout or a log file is the data could get quite large.   the telegraf file output does have  a means to limit file size and rotate logs, keeping N number of logs.